packages/runtime/container-runtime/src/gc/gcDefinitions.ts

```diff
@@ -68,17 +54,11 @@ export const gcGenerationOptionName = "gcGeneration";
 export const gcTestModeKey = "Fluid.GarbageCollection.GCTestMode";
 /** Config key to expire a session after a set period of time. Defaults to true. */
 export const runSessionExpiryKey = "Fluid.GarbageCollection.RunSessionExpiry";
-/** Config key to disable the tombstone feature, i.e., tombstone information is not read / written into summary. */
-export const disableTombstoneKey = "Fluid.GarbageCollection.DisableTombstone";
 /** Config key to override throwing an error when tombstone object is loaded (requested). */
 export const throwOnTombstoneLoadOverrideKey =
 	"Fluid.GarbageCollection.ThrowOnTombstoneLoadOverride";
-/** Config key to enable throwing an error when tombstone object is used (e.g. outgoing or incoming ops). */
-export const throwOnTombstoneUsageKey = "Fluid.GarbageCollection.ThrowOnTombstoneUsage";
 /** Config key to enable GC version upgrade. */
 export const gcVersionUpgradeToV4Key = "Fluid.GarbageCollection.GCVersionUpgradeToV4";
-/** Config key to disable GC sweep for datastores. They'll merely be Tombstoned. */
-export const disableDatastoreSweepKey = "Fluid.GarbageCollection.DisableDataStoreSweep";
 /** Config key to disable auto-recovery mechanism that protects Tombstones that are loaded from being swept (use true) */
 export const disableAutoRecoveryKey = "Fluid.GarbageCollection.DisableAutoRecovery";
```

Comment:
In similar veins (of my another comment), why do we allow disabling auto-recovery? 
> AFAIU, auto-recovery makes GC less destructive; makes Sweep safer (and conservative)! isn't it?

Thoughts? CC: @markfields

Reply:
I agree with this. I will do it separately.